src/api/__test__/getRequestHandler.test.ts

```diff
@@ -0,0 +1,23 @@
+import getRequestHandler from '../getRequestHandler';
+
+describe('get request handler should work as expected', () => {
+  it('should fire callback function when data successfully fetched from API', async () => {
+    const url = `${process.env.REACT_APP_API || ''}/api/v2/DataItemCategoryTree`;
```

Comment:
url is the same and should be defined at the top to remove duplication

Reply:
The urls are actually different, this is what I'm currently using to return an unsuccessful API call, but I probably need to mock this properly?

Reply:
Sorry I meant the ${process.env.REACT_APP_API || ''} part can be extracted